models/topics.model.js

```diff
@@ -0,0 +1,5 @@
+const db = require('../db/connection')
+
+exports.getTopicsData = () => {
+  return db.query('SELECT * FROM topics')
```

Comment:
You'll thank me when you get to front end, but would be better if you got the array, then sent it back, instead of having to access the rows key in the response - but ultimately up to you.